oximeter/db/src/bin/oxdb/main.rs

```diff
@@ -370,10 +360,6 @@ async fn main() -> anyhow::Result<()> {
         Subcommand::Sql { opts } => {
             crate::sql::sql_shell(args.address, args.port, log, opts).await?
         }
-        #[cfg(feature = "oxql")]
-        Subcommand::Oxql { opts } => {
-            crate::oxql::oxql_shell(args.address, args.port, log, opts).await?
```

Comment:
I think we should restructure this PR a bit. As it is, we've sort of spread out the OxQL bits between `oximeter-db` and `omdb`. For example, the `prepare_columns` method is now public so it can be used in `omdb`, but that's really a detail of the OxQL shell implementation.

I feel like it might make more sense to make this particular method public, along with any argument types it needs. (The options type I think). `omdb`'s new `oxql` subcommand would simply call this directly. I think we could _keep_ the `oxdb oxql` subcommand, and really the entirety of the `oximeter-db` crate untouched, and just consume it via the new `omdb oxql` subcommand.

Keeping the implementation in this crate has a few benefits for testing and building (no need to compile all of Omicron), and limits the sprawl of the OxQL-related code.

Reply:
Yep. can definitely do.

Reply:
Though, I think this beckons moving oxql_shell into the src, away from the bin. It feels like an anti-pattern to rely on a bin's function, as it's a lib call made by two bins.

Reply:
I've now moved `prepare_columns` to operate as a fn off the timeseriesschema.

Reply:
Updated as part of moving things to the shells module.